Airports/EGNM/SMR/Labels.txt

```diff
@@ -23,27 +23,30 @@
 "N4" N053.52.04.044 W001.39.09.649 standHold
 ;Stands
 "1" N053.52.10.488 W001.39.42.804 standHold
-"3" N053.52.08.292 W001.39.39.564 standHold
-"5" N053.52.07.212 W001.39.36.936 standHold
-"6" N053.52.07.212 W001.39.35.280 standHold
-"7" N053.52.07.212 W001.39.32.796 standHold
-"8" N053.52.08.000 W001.39.30.925 standHold
-"9" N053.52.08.000 W001.39.28.655 standHold
-"10" N053.52.08.000 W001.39.26.315 standHold
-"11" N053.52.07.968 W001.39.23.364 standHold
-"12" N053.52.07.932 W001.39.20.736 standHold
-"13" N053.52.07.896 W001.39.17.892 standHold
-"14" N053.52.07.896 W001.39.15.372 standHold
-"14A" N053.52.08.100 W001.39.14.372 standHold
-"15" N053.52.07.824 W001.39.12.852 standHold
-"16L" N053.52.07.474 W001.39.10.975 standHold
-"16R" N053.52.07.824 W001.39.10.296 standHold
-"17" N053.52.07.788 W001.39.07.812 standHold
-"18L" N053.52.07.767 W001.39.05.702 standHold
-"18R" N053.52.07.412 W001.39.05.242 standHold
-"19" N053.52.03.090 W001.39.01.230 standHold
-"20A" N053.52.01.682 W001.39.01.412 standHold
-"20B" N053.52.01.202 W001.39.03.774 standHold
-"21A" N053.51.59.883 W001.39.06.103 standHold
-"21B" N053.51.59.243 W001.39.06.169 standHold
-"21C" N053.51.59.123 W001.39.09.330 standHold
+"1L" N053.52.11.480 W001.39.44.380 standHold
+"1R" N053.52.10.890 W001.39.43.490 standHold
+"3" N053.52.09.200 W001.39.39.820 standHold
```

Comment:
A bit out of place. 
![default](https://user-images.githubusercontent.com/18355854/29839661-b2ce0538-8d07-11e7-98ba-49adaf2c2aaa.png)